packages/expo-blob/src/ExpoBlob.types.ts

```diff
@@ -0,0 +1,79 @@
+import { NativeModule, SharedObject } from 'expo';
+
+export declare class Blob {
+  /**
+   * The size of the Blob in bytes.
+   */
+  readonly size: number;
+  /**
+   * The MIME type of the Blob, or the empty string if the type cannot be determined.
+   */
+  readonly type: string;
+
+  /**
+   * Creates a new Blob object from the given parts and options.
+   * @param blobParts An array of BlobPart to include in the Blob.
+   * @param options An optional BlobPropertyBag dictionary.
+   */
+  constructor(blobParts?: BlobPart[], options?: BlobPropertyBag);
+
+  /**
+   * @param start The starting byte index (inclusive) represented as a signed 32 bit integer (up to 2^31 - 1).
+   * @param end The ending byte index (exclusive) represented as a signed 32 bit integer (up to 2^31 - 1).
+   * @param contentType The MIME type of the new Blob. If not provided, defaults to an empty string.
+   * @returns A new Blob object containing the data in the specified range of bytes of the source Blob.
+   */
+  slice(start?: number, end?: number, contentType?: string): Blob;
+
+  /**
+   * Returns a Promise that resolves with the entire contents of the Blob as a Uint8Array.
+   * @returns Promise resolving to the Blob's binary data as a Uint8Array.
+   */
```

Comment:
Can you also update rest of the return statements in this file and the `utils.ts` file (where they are duplicated)?